owlbot.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 Google LLC
```

Comment:
Make this a git mv or fix the year?

Reply:
Apparently git does not fully support this so we'll keep the year from the original synth.py.